client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/hooks/useDraftStocktakeLines.ts

```diff
@@ -8,7 +8,7 @@ export const useDraftStocktakeLines = (
 ): [DraftStocktakeLine[], Dispatch<SetStateAction<DraftStocktakeLine[]>>] => {
   const { id } = useStocktake.document.fields('id');
   const { data: stockLines } = useStockLines(item?.id || '');
-  const { lines } = useStocktake.line.rows();
+  const { lines } = useStocktake.line.rows(item?.code);
```

Comment:
Using itemCode here as theres isn't an item_id filter in the backend yet.
Should I add it? Or is there a better overall solution?

Reply:
There's a risk that we item/code filter an item name might match a code or visa-versa, so I think correct would be to use the item_id